packages/cli/__tests__/zostso/__unit__/receive/app/ReceiveASApp.handler.unit.test.ts

```diff
@@ -0,0 +1,65 @@
+/*
+* This program and the accompanying materials are made available under the terms of the
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at
+* https://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Copyright Contributors to the Zowe Project.
+*
+*/
+
+import { ZosmfRestClient } from "@zowe/core-for-zowe-sdk";
+import { ReceiveTsoApp } from "@zowe/zos-tso-for-zowe-sdk";
+import * as ReceiveASAppHandler from "../../../../../src/zostso/receive/app/ReceiveASApp.handler";
+import * as ReceiveASAppDefinition from "../../../../../src/zostso/receive/app/ReceiveASApp.definition";
+import { IHandlerParameters } from "@zowe/imperative";
+import { mockHandlerParameters } from "@zowe/cli-test-utils";
+import { UNIT_TEST_ZOSMF_PROF_OPTS } from "../../../../../../../__tests__/__src__/TestConstants";
+
+const DEFAULT_PARAMETERS: IHandlerParameters = mockHandlerParameters({
```

Comment:
I understand that we are taking advantage of the cli-test-utils here, but there are instances were taking snapshots is not desirable since it could lead to sensitive information being captured 😓 

That was a general observation, and we may not have much control over what we should consider sensitive data when it comes to tests 😅 

Will discuss further offline 😋